Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -284,8 +284,8 @@ extension _ChatChannelListVC: _ChatChannelListControllerDelegate {
 
         collectionView.performBatchUpdates(
             {
-                collectionView.insertItems(at: Array(insertIndexes))
                 collectionView.deleteItems(at: Array(removeIndexes))
+                collectionView.insertItems(at: Array(insertIndexes))
                 collectionView.reloadItems(at: Array(updateIndexes))
```

Comment:
IMO this doesn't matter and the collection view just performs the operations in the right order after the batch updates closure is finished. But who knows what happens there? 🤷‍♂️

Reply:
Yes maybe you are right, but since IGKitList has the updates with this order, I guess it's better to be sage than sorry 👍